### ISSUE TITLE
CORS issue?

### ISSUE THREAD

**Issue by DougAnderson444** [beakerbrowser/dat-pinning-service-client#2]
@RangerMauve or anybody else hit a CORS issue using this? Trying to reach hashbase.io using this pinning service, hit this wall right away:

> blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.

[https://github.com/beakerbrowser/hashbase/issues/118](https://github.com/beakerbrowser/hashbase/issues/118)

**Comment by RangerMauve** [comment]
Yeah, I think I brought this up with @pfrazee before. You'll need to host your own pinning service that's got the appropriate CORS headers set.

**Comment by DougAnderson444** [comment]
...understood.

**Comment by RangerMauve** [comment]
Shouldn't be too hard to add the headers to Hashbase, I think.

**Comment by DougAnderson444** [comment]
I agree. Been chatting with Paul about it, I'm going to try to do some testing to see if there are any issues with cookies and credentials with the cross origin before committing the PR.

**Comment by RangerMauve** [comment]
Oh hell yeah. Is it going to make it into the official hashbase service? Or is it just a flag that people can set on their own versions?

**Comment by pfrazee** [comment]
We'll add it to the official instance

**Comment by RangerMauve** [comment]
OMG! That's wonderful! It opens the door for so many possibilities! 😍 I'll double check that dat-store has the proper CORS headers too.
